db/dataset_eval.py

```diff
@@ -296,6 +296,28 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
     return job_id
 
 
+def get_user_pending_jobs(user_api_key):
+    with db.engine.connect() as connection:
+        query = sqlalchemy.text("""
+                    SELECT dataset_eval_jobs.id
+                    FROM   dataset_eval_jobs, dataset_snapshot, dataset, api_key
```

Comment:
In postgres there is no difference between the two, but in AB we use `JOIN` instead of `FROM`/`WHERE`, so you should do
```
 SELECT ...
   FROM dataset_eval_jobs
   JOIN dataset_snapshot
     ON dataset_snapshot.id = dataset_eval_jobs.snapshot_id
   JOIN dataset
     ON dataset.id = dataset_snapshot.dataset_id
   .. etc
```